src/Symfony/Bundle/FrameworkBundle/Command/WorkerListCommand.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class WorkerListCommand extends ContainerAwareCommand
+{
+    protected function configure()
+    {
+        $this
+            ->setName('worker:list')
+            ->setDescription('List available workers.')
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $workers = $this->getContainer()->getParameter('worker.workers');
+
+        foreach ($workers as $name => $_) {
```

Comment:
I guess you are referring to the fact this command should be a service and so we could inject only what we need?